src/libraries/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -626,7 +626,7 @@ private bool AcquireClientCredentials(ref byte[] thumbPrint)
         //
         // Acquire Server Side Certificate information and set it on the class.
         //
-        private bool AcquireServerCredentials(ref byte[] thumbPrint, byte[] clientHello)
+        private bool AcquireServerCredentials(ref byte[] thumbPrint, byte[] clientHello, int offset, int count)
```

Comment:
Can this be changed to just take spans instead, or does it need to take the array/offset/count?

Reply:
I will take look. I had to add offset/count to get SNI working properly. But that uses span under the cover anyway.

Reply:
updated.